src/homework/HomeWork.kt

```diff
@@ -17,3 +17,17 @@ Programming exercises:
      * def run() -> prints dog is running, make isHungry to true
      * def eat() -> if Hungry, print Dog eats, if not hungry, print Dog refuses to eat.
  */
+
+fun printFib(until: Int) {
+    val until = 100
```

Comment:
Remember the parameter you have defined until is a variable.  So you don't have to define it.

In fact, if I call ```printFib(10)``` it'll always go up to 100